package.json

```diff
@@ -12,7 +12,8 @@
     "eslint": "^7.3.1",
     "eslint-config-google": "^0.14.0",
     "flags": "^0.1.3",
-    "mocha": "^8.0.1"
+    "mocha": "^8.0.1",
+    "posthtml": "^0.13.1"
```

Comment:
posthtml is a dependency, not a dev dependency.

Also, do we still need tmp? If only in tests, then it's a dev dependency.

Reply:
I need to research the difference! Thanks for bringing this up.